ragas/src/ragas/experimental/metrics/discrete.py

```diff
@@ -23,7 +23,7 @@ def __post_init__(self):
         )
 
     def get_correlation(
-        self, gold_labels: t.List[str], predictions: t.List[str]
+        self, gold_label: t.List[str], predictions: t.List[str]
```

Comment:
why labels -> label?

Reply:
feels out of convention

Reply:
Yes, I called out earlier. This is spurious. Don't need to change this.